app/javascript/dashboard/components-next/input/Input.vue

```diff
@@ -42,9 +42,22 @@ const props = defineProps({
     type: String,
     default: '',
   },
+  autofocus: {
+    type: Boolean,
+    default: false,
+  },
```

Comment:
Instead of adding each of these manually as a prop. We could use disable fall through attributes and add them to the input directly. See: https://vuejs.org/guide/components/attrs#disabling-attribute-inheritance

That way, all the events will also get managed automatically